src/sentry/utils/raven.py

```diff
@@ -57,24 +87,18 @@ def send(self, **kwargs):
             super_kwargs['tags']['install-id'] = options.get('sentry:install-id')
             super(SentryInternalClient, self).send(**super_kwargs)
 
+        key = self.project_key
+        if key is None:
+            return
+
         if not is_current_event_safe():
```

Comment:
interestingly we call this in both capture() and send(), and I'm not really sure if we need both (but I also dont have the time or care to investigate if we can remove one)